docs/integration/social-networks.md

```diff
@@ -8,28 +8,34 @@ You can set up which social network logins are enabled on which domain in `socia
 
 For every social network, you need `app_id` and `app_secret`. Below you have information on where and how to get them. The credentials then need to be set to the corresponding environment variables.
 
+Every social network require URL for backward redirect to you app.
```

Comment:
```suggestion
Every social network require URL for backward redirect to the eshop.
```

or maybe `to the app`? Or maybe it's some form of the sentence I'm not aware of 🤔